src/prefect/client/orion.py

```diff
@@ -96,6 +97,17 @@ def __init__(
     ) -> None:
         httpx_settings = httpx_settings.copy() if httpx_settings else {}
         httpx_settings.setdefault("headers", {})
+        verify = os.getenv("PREFECT_API_TLS_INSECURE_SKIP_VERIFY")
```

Comment:
You can actually add a setting to `prefect.settings` instead! Settings can be configured with environment variables or via or profiles. It'll also take care of casting input to a bool.